reference/java-api.md

```diff
@@ -728,6 +728,8 @@ The following tables describe the Java classes and their methods.
 | &nbsp;&nbsp; public [Node](#java_node) [getFromDef](supervisor.md#wb_supervisor_node_get_from_def)(String name);                                              |
 | &nbsp;&nbsp; public [Node](#java_node) [getFromId](supervisor.md#wb_supervisor_node_get_from_def)(int id);                                                    |
 | &nbsp;&nbsp; public void [setLabel](supervisor.md#wb_supervisor_set_label)(int id, String label, double xpos, double ypos,                                    |
+| &nbsp;&nbsp;&nbsp;&nbsp; double size, int color, double transparency, String font);                                                                           |
+| &nbsp;&nbsp; public void [setLabel](supervisor.md#wb_supervisor_set_label)(int id, String label, double xpos, double ypos,                                    |
```

Comment:
Yes, this is how swig handle default argument in Java, it creates 2 functions, 1 with the argument and another without.